action/src/app.ts

```diff
@@ -128,6 +134,35 @@ async function run(stoatConfig: any) {
   core.info('Uploading workflow outputs...');
   const stoatConfigFileId = await uploadWorkflowOutputs(typedStoatConfig, commentTemplate, githubActionRun);
   await runPlugins(typedStoatConfig, githubActionRun, stoatConfigFileId);
+
+  core.info('Checking Kubernetes plugin...');
+  const namespaces = core.getInput("kubernetes").split(",");
+
+  if(namespaces.length > 0) {
+    const kc = new k8s.KubeConfig();
+    kc.loadFromDefault();
+    const kube = kc.makeApiClient(k8s.CoreV1Api);
+
+    kube.listNamespacedPod()
+        .then((res :any ) => {
+          // tslint:disable-next-line:no-console
+          console.log(res.body);
+        });
+
+    // todo: check if you can connect first
+
+    const logPath = await fs.promises.mkdtemp('/tmp/');
+
+    // then go through all the pods and collect paths generated
+    for (const namespace of namespaces) {
+      const pods = await kube.api.v1.namespaces(namespace).get().pods.get();
+      // download logs
+    }
+
+    // generate index.html file
+
+    await runStaticHostingPlugin("kubernetes-logs", { metadata: { name: "Kubernetes Logs"}, path: logPath } as StaticHostingPlugin, githubActionRun, stoatConfigFileId);
+  }
```

Comment:
It feels that the above logic should be moved to a new plugin?

Reply:
I was thinking of having "file_viewer" be an option for `static_hosting`, not its own plugin.

Reply:
Then this can be moved to the static hosting plugin? I'd prefer not to have multiple special cases in the main `app`.